### ISSUE TITLE
Using add_imports makes isort unusable

### ISSUE THREAD

**Issue by ziima** [timothycrosley/isort#1297]
Since the release of isort 5, the `add_imports` options stopped working.

Let's have a file with only imports:
```python
from __future__ import unicode_literals

from os import path
```

Running
```
isort --check-only --diff test.py --add-import 'from __future__ import unicode_literals'
```
results in
```
ERROR: /tmp/test/test.py Imports are incorrectly sorted and/or formatted.
```
with no diff provided.

**Comment by timothycrosley** [comment]
Thanks for reporting this issue! The combination of checking while adding imports wasn't previously tested. I've fixed the issue in 5.0.5 and added an additional regression test to ensure it doesn't resurface.

Thanks!

~Timothy

**Comment by ziima** [comment]
I confirm it works with 5.0.5. Thanks
